src/app/modules/main/wallet_section/activity/controller.nim

```diff
@@ -277,10 +277,16 @@ QtObject:
             return
 
           controller.processResponse(responseJson)
+        of transactions.eventNewTransfers:
```

Comment:
I pick up these changes in this PR: https://github.com/status-im/status-desktop/pull/11411 not to wait for status-go merging.